### DISCUSSION TITLE
Search for a subset of people

### DISCUSSION THREAD

**Discussion by frankasphaug** [microsoft-search/pnp-modern-search#1851] [General]
I'm trying to search for a subset of people in the Verticals webpart.
As a default People has  
   "Result Source Identifier" = b09a7990-05ea-4af9-81ef-edfab16c4e31
   "Query template" = {searchTerms}

How can I edit "Query Template" so that I show people from a specific company?

**Comment by FranckyC** [comment]
If I'm correct, there is no builtin property for 'Company'. It means you should use a custom user profile property as set it as 'indexed'. Then map the generated crawled property to a _RefinableString_ managed property so it can be used in the query template like this:

`RefinableStringXX:{verticals.value}`

Where each vertical has the company name as tab value.
